### ISSUE TITLE
no-conditional-in-test rule triggers for test metadata conditionals (false-positive)

### ISSUE THREAD

**Issue by touchedthecode** [playwright-community/eslint-plugin-playwright#363]
When using a conditional in the test function's metadata parameters, the no-conditional-in-test rule is triggered, even though the conditional is outside the actual test body.

```js
test('My Test', { tag: productType === 'XYZ' ? '@regression' : '@smoke' }, () => {
  // test code
})
```

I would expect this rule to only apply to conditionals inside the test body ({}), not in the metadata.

It's not a big deal, as I can work around it by assigning the value to a variable beforehand. Just wanted to raise this in case it's worth adjusting the rule behavior.
